### ISSUE TITLE
Add a keras.io example using OPT

### ISSUE THREAD

**Issue by mattdangerw** [keras-team/keras-nlp#923]
Need a little planning to come up with a good example here, but we should ship some sort of keras.io example using OPT.

**Comment by mattdangerw** [comment]
One option I think would be legitimately exciting would be to show fine-tuning the dataset released with the alpaca model: https://crfm.stanford.edu/2023/03/13/alpaca.html

**Comment by amangupta201** [comment]
One can be by using compile() & fit():
from tensorflow import keras
from tensorflow.keras import layers

model = keras.Sequential()
model.add(layers.Dense(64, kernel_initializer='uniform', input_shape=(10,)))
model.add(layers.Activation('softmax'))

opt = keras.optimizers.Adam(learning_rate=0.01)
model.compile(loss='categorical_crossentropy', optimizer=opt)
or
model.compile(loss='categorical_crossentropy', optimizer='adam')
